File: examples/tutorial/hello/go/func.go
package main

import (
	"encoding/json"
	"fmt"
	"log"
	"os"
)

type person struct {
	Name string
}

func main() {
	p := &person{Name: "World"}
	json.NewDecoder(os.Stdin).Decode(p)
	mapD := map[string]string{"message": fmt.Sprintf("Hello %s", p.Name)}
	mapB, _ := json.Marshal(mapD)
	fmt.Println(string(mapB))

	log.Println("---> stderr goes to the server logs.")
}

File: examples/tutorial/hello/java/Func.java
import java.io.*;

public class Func {
    public static void main(String[] args) throws IOException {
        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));

        String name = bufferedReader.readLine();
        name = (name == null) ? "world"  : name;

        System.out.println("Hello, " + name + "!");
    }

}
